java_tests/src/main/java/org/infinispan/client/hotrod/impl/RemoteCacheImpl.java

```diff
@@ -354,6 +354,72 @@ public void clear() {
         }
    }
 
+   private cli.Infinispan.HotRod.Wrappers.DictionaryOfObjects mapToDict(Map<? extends K, ? extends V> m)
+   {
+	   cli.Infinispan.HotRod.Wrappers.DictionaryOfObjects dict = new cli.Infinispan.HotRod.Wrappers.DictionaryOfObjects();
+       for (Entry<? extends K, ? extends V> entry : m.entrySet()) {
+           dict.Add(marshal(entry.getKey()), marshal(entry.getValue()));
+       }
+       return dict;
+   }
+   
+   @Override
+   public CompletableFuture<Void> putAllAsync(Map<? extends K, ? extends V> map) {
+       try {
+           if (false) workaroundCLICheckedExceptions();
```

Comment:
I don't understand this "if (false) ...." Will this ever be called? What's the purpose?